Gems/Atom/RPI/Code/Tests/Model/ModelTests.cpp

```diff
@@ -989,6 +990,9 @@ namespace UnitTest
         uint32_t{ 0 }, 2, 1, 1, 2, 3, 4, 5, 6, 5, 7, 6, 0, 4, 2, 4, 6, 2, 1, 3, 5, 5, 3, 7, 0, 1, 4, 4, 1, 5, 2, 6, 3, 6, 7, 3,
     };
 
+    static constexpr AZStd::array QuadPositions = { -1.0f, 1.0f, 0.0f, 1.0f, 1.0f, 0.0f, -1.0f, -1.0f, 0.0f, 1.0f, -1.0f, 0.0f };
+    static constexpr AZStd::array QuadIndices = { uint32_t{ 0 }, 2, 1, 1, 2, 3 };
+
```

Comment:
Do these quads make a cube? I think these could use a better name.

Reply:
Nope just a quad 🙂 I just wanted a valid mesh that was different to the Cube above